sphinx_js/__init__.py

```diff
@@ -9,6 +10,68 @@
 from .jsdoc import Analyzer as JsAnalyzer
 from .typedoc import Analyzer as TsAnalyzer
 
+
+# Cache this to guarantee it only runs once.
+@lru_cache(maxsize=None)
```

Comment:
I figured I'd guard against this running multiple times even though "normal use" probably won't run this multiple times. I added the comment because this is a frankly bizarre use of `lru_cache`.